models/workoutlog.js

```diff
@@ -6,19 +6,18 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const exerSchema = new mongoose.Schema({
-    exercise: {
-        type: String,
-        enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
-    }
+    exercise: String,
+        // enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
+    
 });
 
-const findExercise = module.exports = mongoose.model('exercise', exerSchema);
+module.exports = mongoose.model('Exercise', exerSchema);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
module.exports = mongoose.model("Exercise", exerSchema);
```